internal/sys/password_policy.go

```diff
@@ -0,0 +1,252 @@
+package sys
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/base"
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/client"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
```

Comment:
Would it be beneficial to have some compile-time check to ensure we're implementing the right interfaces? 

Something like:
```go 
var _ resource.ResourceWithConfigure = &PasswordPolicyResource{}
```